### DISCUSSION TITLE
What causes left side to be double image and right side of photo is ok.

### DISCUSSION THREAD

**Discussion by Moikka2** [arkypita/LaserGRBL#2177] [Q&A]
Please see photo of problem, the star is double for the first 3 and 2 are ok. this is constant.
- changed starting position but no effect. 
- changed to slow speed / slow buffer / slow linear speed -> no effect. 

What could be the problem 
(Laser = Sculpfun s9) (Laser GRBL 5.7.0) 
To my undertsanding this started when 5.7 was installed. 
![Näyttökuva 2024-01-09 162137](https://github.com/arkypita/LaserGRBL/assets/156098387/9e2ee402-f48e-442f-a90f-1a51a79fa525)
and secondary question. always this has happened but so small -> the most left side has always lost the top spike part (when the picture is ok, it also looses then)

Any advice what to test?
![Näyttökuva 2024-01-09 162211](https://github.com/arkypita/LaserGRBL/assets/156098387/d346b487-73b6-4a1f-9c63-5b630256b416)

**Comment by Moikka2** [comment]
Update. reverted back to 5.6.0 and all ok. so this is 5.7 bug? (how to report official bug?)

**Reply by arkypita** [reply]
Nothing has changed between 5.6 and 5.7 that can cause your issue.
I suggest you do some further testing before claiming otherwise.

In my opinion the problem must be looked for elsewhere.

**Comment by Moikka2** [comment]
HiYes this is odd because tested all wires. The belts etc. Cleaned nozzle. Nothing else helped than reverting to old. Might there be setting change between versions?Sent from Android deviceOn 9 Jan 2024 19.18, arkypita ***@***.***> wrote:
Nothing has changed between 5.6 and 5.7 that can cause your issue.
I suggest you do some further testing before claiming otherwise.
In my opinion the problem must be looked for elsewhere.
